settings.gradle.kts

```diff
@@ -0,0 +1,12 @@
+include(":app")
+
+val enableDependencySubstitution = true
```

Comment:
Could we make this check an environment variable?

Reply:
My initial idea was to add a property that could be added in the local.properties but I didn't get it working properly. Environment variable might be an idea too.

Reply:
Added this one now. Adding `enable.dependency.substitution=false` to the `local.properties` file will disable dependency substitution. It's enabled by default.